### PR TITLE
EX4-6

### PR DESCRIPTION
**Pull request by contingency40**
This PR contains the following:

- Dialogue when caught by High Slug in the chase scene
- "Puzzle" note
- Pre and post battle dialogue with High Lamia when Marion appears
- Marion's skills and skill battle text
- Marion's menus
- Right-aligned Luka's menus (Anegorami's two new plugins required but not included in this PR)
- Mizuki's two support skills
- The intro to the tea party up until the save point
- A few misc fix-ups here and there

Shrifted:
- Added plugin files
- Added a bunch of edge stuff around Marion's equipment

### CODE REVIEW

File: Gaijinizer patch/Armors.txt [hunk 1]
```diff
@@ -1001,29 +1001,29 @@ Cloth Clothes#StrateUnsure
 > BEGIN STRING
 マリオンの世界に自生している赤リンゴ
 \\C[5]マリオンの回復スキルの対象が味方全体になる。
-> CONTEXT: Armors/101/description
-
+> CONTEXT: Armors/101/description < UNTRANSLATED
+ADD STUFF HERE
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Armors.txt:1006]: There's a few entries in this file that cover descriptions for the Apples you'll need to cover.

File: Gaijinizer patch/Map468.txt [hunk 2]
```diff
@@ -3,360 +3,431 @@
...
-> CONTEXT: Map468/events/46/pages/1/17/Dialogue < UNTRANSLATED
-> CONTEXT: Map541/events/46/pages/1/17/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map468/events/46/pages/1/17/Dialogue
+> CONTEXT: Map541/events/46/pages/1/17/Dialogue
+The angels declared that my family was to be sent into
+exile from this place as punishment. This decree was made
+without even knowing the purpose of this estate built as a
+foothold during the monster subjugation.#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map468.txt:342]: Note

File: Gaijinizer patch/Map468.txt [hunk 3]
```diff
@@ -3,360 +3,431 @@
...
-> CONTEXT: Map468/events/46/pages/1/23/Dialogue < UNTRANSLATED
-> CONTEXT: Map541/events/46/pages/1/23/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map468/events/46/pages/1/23/Dialogue
+> CONTEXT: Map541/events/46/pages/1/23/Dialogue
+It would seem that as the ages passed, the foolish angels
+themselves had forgotten about the being they had once
+ordered us to seal away. The duty thus not being passed on
+is a result befitting their dimwittedness.#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map468.txt:355]: Note

File: Gaijinizer patch/Map468.txt [hunk 4]
```diff
@@ -3,360 +3,431 @@
...
-> CONTEXT: Map468/events/46/pages/1/41/Dialogue < UNTRANSLATED
-> CONTEXT: Map541/events/46/pages/1/41/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map468/events/46/pages/1/41/Dialogue
+> CONTEXT: Map541/events/46/pages/1/41/Dialogue
+Though they took everything from me, this piece of armor
+is the one thing I will not let them have, so I've hidden
+it here. This armor of my predecessors is the only thing
+I must never lose.#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map468.txt:394]: If we want it to sound *really* old, I'd put "forefathers" over "predecessors".

File: Gaijinizer patch/Map470.txt [hunk 5]
```diff
@@ -1,181 +1,227 @@
...
 本当は、もっと豪華なものを用意したかったんだけど
 隠密性を重視した結果、無駄な装飾を省いたんだ。
-> CONTEXT: Map470/events/1/pages/0/23/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map470/events/1/pages/0/23/Dialogue/enemy27
+Most apologies for providing such a modest
+welcome, I did plan on preparing something more
+lavish but with the need to remain hidden, I had
+to cut back on pointless decorations.#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map470.txt:36]: Minor change:
```
My apologies for providing such a modest
welcome... I did plan on preparing something
more lavish, but I had to cut back on pointless
decorations to be a bit more discrete.
```

File: Gaijinizer patch/Map470.txt [hunk 6]
```diff
@@ -1,181 +1,227 @@
...
 座ってお茶でも飲みながら
 順を追って説明していこうじゃないか………
-> CONTEXT: Map470/events/1/pages/0/55/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map470/events/1/pages/0/55/Dialogue/enemy27
+You've all gone to the trouble of gathering here
+from each of your worlds. Why don't you all sit,
+have some tea or something, and I'll explain
+everything step-by-step.#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map470.txt:106]: Revision for flow:
```
You've all gone through the effort of coming here
from each of your worlds. Take a seat,
enjoy a fresh cup of tea, and I'll explain it
all, piece by piece...
```

File: Gaijinizer patch/Map470.txt [hunk 7]
```diff
@@ -1,181 +1,227 @@
...
 \\N[1]と声が同じように思えるのだが…
 
 ヤツはどういう魔物なのだ…？
-> CONTEXT: Map470/events/1/pages/0/106/Dialogue/enemy35 < UNTRANSLATED
-
+> CONTEXT: Map470/events/1/pages/0/106/Dialogue/enemy35
+In fact, it seems both \\N[1] and her have
+rather similar voices. Just what kind of monster
+is she exactly?#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map470.txt:177]: ellipses after "voices".

File: Gaijinizer patch/Map470.txt [hunk 8]
```diff
@@ -1,181 +1,227 @@
...
 
 そもそも、アイツって生命体なの……？
-> CONTEXT: Map470/events/1/pages/0/111/Dialogue/enemy38 < UNTRANSLATED
-
+> CONTEXT: Map470/events/1/pages/0/111/Dialogue/enemy38
+No, my summoning device is giving some really
+buggy responses, so it's hard to tell whether
+she's a monster or a demon. In fact, is she
+even a living organism?#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map470.txt:189]: Rephrasing for flow (let me know if this is a bit too far from the original text).
```
No... my summoner is throwing a bunch of errors.
It's hard to tell whether she's even a monster
or a demon... Is... Is she even a living being in
the first place...?
```

File: Gaijinizer patch/Map469.txt [hunk 9]
```diff
@@ -3,721 +3,912 @@
...
 
 この姿では、いくら子供に戻った相手といえど
 人一人抱えて走り回るのは、ちょっとな………
-> CONTEXT: Map469/events/3/pages/0/115/Dialogue/alice_fc5 < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/115/Dialogue/alice_fc5
+If he were in his adult form we'd at least have
+our powers back, but carrying him around even
+in this child form would be a little too much...#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:66]: minor revision for flow.
```
If he were an adult we'd have our powers back.
But right now even carrying him around while
he's a child would be a bit too much...
```

File: Gaijinizer patch/Map469.txt [hunk 10]
```diff
@@ -3,721 +3,912 @@
...
 > BEGIN STRING
 ……………あれ？
 
 これ、私が言うべきだったのかしら……？
-> CONTEXT: Map469/events/3/pages/0/130/Dialogue/enemy38 < UNTRANSLATED
+> CONTEXT: Map469/events/3/pages/0/130/Dialogue/enemy38
+...Huh?
 
+Am I supposed to chime in on this?#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:88]: 6 ellipses here.

File: Gaijinizer patch/Map469.txt [hunk 11]
```diff
@@ -3,721 +3,912 @@
...
 魔物の大群に立ち向かう、人間代表の朴念仁！
 \\C[2]その名も『マリオン』！！！
-> CONTEXT: Map469/events/3/pages/0/267/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/267/Dialogue/sirousagi_fc1
+Making full use of his predecessor Saito's angelic
+magic to violently hurl weapons and face down
+hordes of monsters, it's the obstinate
+representative of humanity, Marion!!!#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:227]: Note

File: Gaijinizer patch/Map469.txt [hunk 12]
```diff
@@ -3,721 +3,912 @@
...
 
 > BEGIN STRING
 天使の寵愛を受けた、聖騎士『サイトウ』……
 
 その子孫であるマリオンくんは、かつて天使のオーブを巡る
 天使と魔物の騒乱を解決した人間なんだ。
-> CONTEXT: Map469/events/3/pages/0/412/Dialogue/sirousagi_fc1 < UNTRANSLATED
+> CONTEXT: Map469/events/3/pages/0/412/Dialogue/sirousagi_fc1
+Marion is the descendant of the holy knight
+"Saitou" who had the blessings of the angels.
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:409]: "Saitou" or Saito? Both have been used before this.

File: Gaijinizer patch/Map469.txt [hunk 13]
```diff
@@ -3,721 +3,912 @@
...
 しかし、『無言系』となると
 
 翻訳できるヒロインというか…付き添いが欲しいところだな
-> CONTEXT: Map469/events/3/pages/0/429/Dialogue/alice_fc5 < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/429/Dialogue/alice_fc5
+But if he's the "silent type", then I'd like to
+have someone here who can translate things to
+him for us...#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:447]: Note

File: Gaijinizer patch/Map469.txt [hunk 14]
```diff
@@ -3,721 +3,912 @@
...
 
 まさしくその通りだよ。
-> CONTEXT: Map469/events/3/pages/0/528/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/528/Dialogue/sirousagi_fc1
+\\C[2]When you used the linear cannon on Samael after
+her magic went out of control and he manifested,
+you blew her power into smithereens.\\C[0]
+And that's how it remains.#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:647]: Note

File: Gaijinizer patch/Map469.txt [hunk 15]
```diff
@@ -3,721 +3,912 @@
...
-> CONTEXT: Map469/events/3/pages/0/553/Dialogue/enemy35 < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/553/Dialogue/enemy35
+But then I also heard that though she had fallen
+from the heavens, she still lived side-by-side
+with the humans, keeping them under her protection
+while atoning for her sins.
+Hearing that made me feel, in a good way, like my
+old friend hasn't changed a bit.#RonIffyWording 
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:708]: Note

File: Gaijinizer patch/Map469.txt [hunk 16]
```diff
@@ -3,721 +3,912 @@
...
 > BEGIN STRING
 ほ、本当に知り合いなんだ……
 
 なんていうか…インパクトあるなぁ……
-> CONTEXT: Map469/events/3/pages/0/561/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map469/events/3/pages/0/561/Dialogue/kazuya
+T-They really do know each other.
 
+This is like... really impactful.#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:718]: Note

File: Gaijinizer patch/Map469.txt [hunk 17]
```diff
@@ -3,721 +3,912 @@
...
 ボクの『道案内』も、これで終わり……
 
 後のことは、お茶会の主催者……
 \\C[2]『マッドハッター』\\C[0]に任せるよ。
-> CONTEXT: Map469/events/3/pages/0/600/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/600/Dialogue/sirousagi_fc1
+This as far as I go being your guide, now it's up
+to the organizer of the Tea Party, the \\C[2]Mad Hatter\\C[0].
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:807]: Add attribution.

File: Gaijinizer patch/Map469.txt [hunk 18]
```diff
@@ -3,721 +3,912 @@
...
 
 ボクも慌てたり驚いたり
 慣れないことばっかりで、いい加減疲れたよ……
-> CONTEXT: Map469/events/3/pages/0/595/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/595/Dialogue/sirousagi_fc1
+You're right, I'm sick and tired of being spooked,
+running around everywhere, and dealing with all
+these new things.#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:797]: Note

File: Gaijinizer patch/Map468.txt [hunk 19]
```diff
@@ -3,360 +3,431 @@
...
-> CONTEXT: Map468/events/46/pages/1/11/Dialogue < UNTRANSLATED
-> CONTEXT: Map541/events/46/pages/1/11/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map468/events/46/pages/1/11/Dialogue
+> CONTEXT: Map541/events/46/pages/1/11/Dialogue
+Though our previous generation, with the blessings of the
+angels, fervently carried on their duty of continuing to
+seal away the vicious "vampires"... they suffered such
+appalling mistreatment...#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map468.txt:329]: Looking at the DeepL translation and what the next box says, I think this makes more sense?
```
Our previous generations, with the blessings of the
angels, fervently carried on their duty of continuing to
seal away the vicious "vampires"... 
And *this* is what they repay us with.
```

File: Gaijinizer patch/Map469.txt [hunk 20]
```diff
@@ -3,721 +3,912 @@
...
 
 別に喋れないってワケじゃないけど
 ほら、『無言系』ってやつ？
-> CONTEXT: Map469/events/3/pages/0/389/Dialogue/sirousagi_fc1 < UNTRANSLATED
+> CONTEXT: Map469/events/3/pages/0/389/Dialogue/sirousagi_fc1
+Marion-kun doesn't talk.
 
+It's not that he can't talk, but he's what you'd
+call, you know, the "silent type".#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:351]: Minor revision:
```
Marion-kun doesn't talk.

It's not that he can't talk, but he's what you'd
call... you know... the "silent type".
```

File: Gaijinizer patch/Map469.txt [hunk 21]
```diff
@@ -3,721 +3,912 @@
...
 
 それじゃあ、彼がこの世界で見つける予定だった子？
 随分と年齢は若いように見えるけど……
-> CONTEXT: Map469/events/3/pages/0/402/Dialogue/enemy38 < UNTRANSLATED
+> CONTEXT: Map469/events/3/pages/0/402/Dialogue/enemy38
+You said just before that he's the last guest...
 
+Does that mean he's the one we were planning on
+finding in this world? He looks quite young.#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:387]: remove the "just" here.

File: Gaijinizer patch/Map469.txt [hunk 22]
```diff
@@ -3,721 +3,912 @@
...
 
 \\C[2]『射撃武器を投擲すること』『投擲物を殴打に使うこと』\\C[0]
 で、相手を攻撃すると発動する。
-> CONTEXT: Map469/events/3/pages/0/417/Dialogue/sirousagi_fc1 < UNTRANSLATED
+> CONTEXT: Map469/events/3/pages/0/417/Dialogue/sirousagi_fc1
+His angelic power has a unique manifestation.
 
+He attacks by \\C[2]throwing ranged weapons, and
+bludgeoning with throwing weapons.\\C[0]#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:426]: minor revision for flow:
```
His angelic powers manifest in a unique way.

He attacks by \\C[2]throwing ranged weapons, and
bludgeoning with throwing weapons.\\C[0]
```

File: Gaijinizer patch/Map469.txt [hunk 23]
```diff
@@ -3,721 +3,912 @@
...
 
 \\N[1]くんもよく知る、彼女だよ。
-> CONTEXT: Map469/events/3/pages/0/494/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/494/Dialogue/sirousagi_fc1
+That's right, I was introduced to this world's
+"guest" thanks to the inter-angel connections of
+a certain angel in particular. I think you know
+who I'm talking about, \\N[1]-kun.#RonLocalization
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:577]: Turn the comma after "right" into a period.

File: Gaijinizer patch/Map469.txt [hunk 24]
```diff
@@ -3,721 +3,912 @@
...
 > BEGIN STRING
 ええ…中村。
 
 私も、また顔を合わせることになるとは思わなかったわ……
-> CONTEXT: Map469/events/3/pages/0/511/Dialogue/other5 < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/511/Dialogue/other5
+Yes Nakamura, I also didn't expect us to ever
+meet again.#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:602]: Minor revision here:
```
Yes, Nakamura... I also didn't expect us to ever
meet again.
```

File: Gaijinizer patch/Map469.txt [hunk 25]
```diff
@@ -3,721 +3,912 @@
...
 
 彼女が事前にこっちの世界と話を付けてくれてたんだ。
-> CONTEXT: Map469/events/3/pages/0/543/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map469/events/3/pages/0/543/Dialogue/sirousagi_fc1
+She's lost the power necessary to manifest, this
+ghost-like body is the best she can do. But she
+was able to arrange things ahead of time for us
+in this world.#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map469.txt:682]: change the comma into three ellipses.

File: Gaijinizer patch/Map469.txt (hunk 11 above)
Comment (contingency40) [reply Gaijinizer patch/Map469.txt:227]: Actually now that I've got the benefit of hindsight and know Marion's character slightly better, I think she meant the other meaning of 朴念仁 which is "quiet unsocialable person".

File: Gaijinizer patch/Map469.txt (hunk 12 above)
Comment (contingency40) [reply Gaijinizer patch/Map469.txt:409]: Should be Saito, I'll fix

Comment (Triketos) [reply Gaijinizer patch/Map469.txt:409]: Cool, I'll trust your judgement on this.

File: Gaijinizer patch/Map469.txt (hunk 11 above)
Comment (Triketos) [reply Gaijinizer patch/Map469.txt]: Let's use `aloof` here.

**Review by Triketos — APPROVED** [review]
Looks good to me!

**Pull request merged**